### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by TheLuisAbarca**
## Initial Setup

> In this branch, I have made the following changes:

1. Add basic structure of a react app.
2. Add configuration for linters of React & Redux.
3. Fix linter errors for the default React app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
-# math-magicians
-A calculator made using REACT as frontend technology.
+# Getting Started with Create React App
```
Comment (TedLivist) [inline README.md]: - [x] I would recommend you do away with the dummy **`create-react-app`** texts that do not necessarily relate to the project in any way. I'd suggest you format your README using the Microverse template [here](https://github.com/microverseinc/readme-template)
- [x] In your README remember to add instructions on how users can get your app repo locally, and also to run `npm install` to install packages and `npm start` to start project.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @TheLuisAbarca,

Good job so far!
✔️ Apps runs correctly
✔️ No console log errors
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (TheLuisAbarca) [reply README.md]: Thanks, @TedLivist  for your feedback!, I really appreciate it. Your suggestions have been implemented on commit f154032.

**Comment by omar-labana** [comment]
Hi @TheLuisAbarca, 

Your project is complete!  There is nothing else to say other than... it's time to merge it ╰(*°▽°*)╯
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**